### PR TITLE
Added s3 sync workflow

### PR DESCRIPTION
**Pull request by shoaibtaqi04**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - uses: actions/checkout@v4
+    - uses: kersvers/s3-sync-with-cloudfront-invalidation@v1.0.0
+      with:
+        args: --delete --exclude '.git*/*'
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_S3_BUCKET: din.global
+        DISTRIBUTION_ID: 'EJ9HYRC6B1P88'
+        AWS_REGION: 'us-east-1'   # optional: defaults to us-east-1
```
Comment (ydmk) [inline .github/workflows/push.yml]: use aws s3 cli directly?

Comment (shoaibtaqi04) [reply .github/workflows/push.yml]: ok

Comment (shoaibtaqi04) [reply .github/workflows/push.yml]: done

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_REGION }}
+
+    - name: Sync files to S3 bucket and Invalidate 
+      run: |
+        aws s3 sync ./ s3://din.global --delete --exclude '.git*/*'
```
Comment (ydmk) [inline .github/workflows/push.yml:24]: this is the actual name of the bucket?

din.global??

Comment (shoaibtaqi04) [reply .github/workflows/push.yml:24]: yes. `din.global` is bucket name.

**Review by ydmk — APPROVED** [review]

**Pull request merged**